content-cache/src/nginx_manager.py

```diff
@@ -27,6 +27,7 @@
 logger = logging.getLogger(__name__)
 
 NGINX_CERTIFICATES_PATH = Path("/etc/nginx/certs")
+NGINX_CONF_PATH = Path("/etc/nginx/conf.d")
```

Comment:
I'd suggest NGINX_CONFD_PATH or NGINX_CONF_DIRECTORY_PATH as NGINX_CONF_PATH may be used later for /etc/nginx/nginx.conf

Reply:
Updated.
Thanks for the feedback.